docs/source/config-options.md

```diff
@@ -249,6 +249,10 @@ EnterpriseGatewayApp options
     Default: False
     Is YARN Kerberos/SPNEGO Security enabled (True/False).
     (EG_YARN_ENDPOINT_SECURITY_ENABLED env var)
+--EnterpriseGatewayApp.KernelSessionManager.enable_persistence=<Bool>
```

Comment:
Does this work?  Will this show up when running `juptyer-enterprisegateway --help-all`?

I know that using `KernelSessionManager.enabled_persistence=True` works.  Not sure how this changes things unless I'm missing another change or wasn't aware that internal classes could be appended to the app name for this.